notebooks/02-ingestion/13-native-transforms.ipynb

```diff
@@ -0,0 +1,875 @@
+    "druid.sql.wait_until_ready(table_name, verify_load_status=False)\n",
+    "display.table(f'{table_name}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a8fb66c3-9cd2-4b82-a95b-af47874b8321",
+   "metadata": {},
+   "source": [
+    "Wait for a moment or two. This will allow Druid to apply the new configuration for the ingestion by shutting down the existing task, and starting a new ingestion task with the transforms.\n",
+    "\n",
+    "Then run the query below to see the effect this has had on the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1b6cf5ab-cba3-43d4-a43d-9b095311d05b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import datetime, timedelta\n",
+    "\n",
+    "time_now = datetime.now().strftime('%Y-%m-%dT%H:%M:%S')\n",
+    "print(time_now)\n",
+    "\n",
+    "sql=f'''\n",
+    "SELECT\n",
+    "  \"__time\",\n",
+    "  \"event_type\",\n",
+    "  \"client_ip\",\n",
+    "  \"client_device\",\n",
+    "  \"client_lang\",\n",
+    "  \"client_country\"\n",
+    "FROM \"{table_name}\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'PT15S/{time_now}')\n",
+    "ORDER BY __time DESC\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fb5ceca5-cecc-4710-8092-9aa0f41488af",
+   "metadata": {},
+   "source": [
+    "Since the new ingestion specification continued where the old one finished, the function has only been applied to new data.\n",
+    "\n",
+    "Run the following cell to see the values for `client_country` in data five minutes ago."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "40f55492-3682-4d60-b6c0-0a8d77b8066e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "time_then = (datetime.now() -  timedelta(hours=0, minutes=5)).strftime('%Y-%m-%dT%H:%M:%S')\n",
+    "\n",
+    "sql=f'''\n",
+    "SELECT\n",
+    "  \"__time\",\n",
+    "  \"event_type\",\n",
+    "  \"client_ip\",\n",
+    "  \"client_device\",\n",
+    "  \"client_lang\",\n",
+    "  \"client_country\"\n",
+    "FROM \"{table_name}\"\n",
+    "WHERE TIME_IN_INTERVAL(__time,'PT15S/{time_then}')\n",
+    "ORDER BY __time DESC\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1280251e-2594-45ec-a098-87e2da61be5c",
+   "metadata": {},
+   "source": [
+    "## Use CASE to generate NULL values\n",
+    "\n",
+    "In this section you will use a case function to catch raw data that has an \"unknown\" or \"none\" value. When caught, you will instead write NULL into the data.\n",
+    "\n",
+    "* case_searched() checks if the value of `keyword` is \"None\". If it is, null is returned, otherwise the existing value is returned.\n",
+    "* The same method is then used on `product` and `referrer`.\n",
+    "* In each case, the `name` means that each time, the data from each evaluated dimensions is being overwritten.\n",
```

Comment:
```suggestion
    "* In each case, the `name` means that each time, the data from each evaluated dimensions is being overwritten.\n",
```to 
This sentence is a bit confusing. Is the purpose of this sentence to define the `name` property or to inform the reader that the script overrides each evaluated dimension?